tasks/cephfs/filesystem.py

```diff
@@ -220,6 +219,15 @@ def get_active_names(self):
 
```

Comment:
Hmm?  Were you looking at are_daemons_healthy?  The change is to the get_daemon_names function where it is now comparing with the passed in state.